.github/workflows/main.yml

```diff
@@ -0,0 +1,65 @@
+name: CI
```

Comment:
Let's add some documentation at the top.  [Here](https://github.com/dbt-labs/actions/blob/9690cd8f93cfd99de66a4af00e5b2126ff193945/.github/workflows/changelog-existence.yml#L1-L27) is the general format we use.

Reply:
No, just the documentation.  Please don't add changie 😄 

```markdown
# **what?**
#

# **why?**
...
```